docs/getting-started-guides/docker-multinode.md

```diff
@@ -74,54 +81,146 @@ This pattern is necessary because the `flannel` daemon is responsible for settin
 all of the Docker containers created by Kubernetes.  To achieve this, it must run outside of the _main_ Docker daemon.  However,
 it is still useful to use containers for deployment and management, so we create a simpler _bootstrap_ daemon to achieve this.
 
-You can specify k8s version on very node before install:
+## Starting a Cluster
 
-```
+An example cluster is listed below:
+
+| IP Address  |   Role   |
+|-------------|----------|
+|10.10.102.152|   node   |
+|10.10.102.150|node (and master)|
+
+Optional, you can specify version before deployment, otherwise, we'll use latest `hyperkube` release as default.
+
+```sh
 export K8S_VERSION=<your_k8s_version (e.g. 1.0.3)>
 ```
 
-Otherwise, we'll use latest `hyperkube` image as default k8s version.
+Every machine in `NODES` will be deployed as a Node (Minion), and you need to choose one of the `NODES` as Master, see below:
+
+In `cluster/` directory:
+
+```sh
+export NODES="vcap@10.10.102.150 vcap@10.10.102.152"
+export MASTER="vcap@10.10.102.150"
```

Comment:
kubelet on the master node will still register itself as a node (and accept pod bind) anyway. We need to prevent that (for example, give it a fault master's addr) somehow ...